flink-runtime/src/main/java/org/apache/flink/runtime/security/KerberosUtils.java

```diff
@@ -52,6 +51,13 @@
 
 	private static final AppConfigurationEntry userKerberosAce;
 
+	/* Return the Kerberos login module name */
+	public static String getKrb5LoginModuleName() {
```

Comment:
I assume this was copied from hadoop?

Reply:
Yes, this had a dependency on `KerberosUtil` form Hadoop just for this method. Now we can have Kerberos independent of Hadoop.